tr/lessons/basics/collections.md

```diff
@@ -0,0 +1,184 @@
+---
+version: 1.0.0
```

Comment:
The metadata for lessons has been _greatly_ simplified :tada:, we can update this to just:

```markdown
---
version: 0.9.0
title: Collections
---
```